docs/modules/ROOT/pages/api/erc20.adoc

```diff
@@ -8,7 +8,7 @@
 
 include::../utils/_common.adoc[]
 
-Reference of interfaces and utilities related to ERC20 contracts.
+Reference of interfaces, presets, and utilities related to ERC20 contracts.
```

Comment:
In other doc files we start with "This crate provides....". I think for consistency we should try to do the same or similar on each individual package. WDYT?